### PR TITLE
Improvement to Headless product guide

### PR DESCRIPTION
**Pull request by mnocon**
While reading https://doc.ibexa.co/en/latest/ibexa_products/headless/ I've noticed that the `Contact Us` link is wrong - I've corrected it so it points at the correct page

I've tried to keep the changes to minimum (I like this article a lot!), but I've also made a couple of changes:
1) The PIM section is currently not formatted as a list: https://doc.ibexa.co/en/latest/ibexa_products/headless/#many-ways-to-structure-and-organize-content
2) This is my first encounter with the word `impermeability` - I think `data isolation` is more accessible to the readers

### CODE REVIEW

**Comment by adriendupuis** [comment]
Preview: https://ez-systems-developer-documentation--2409.com.readthedocs.build/en/2409/ibexa_products/headless/

**Review by adriendupuis — APPROVED** [review]
Thank you for fixing the grammar, and simplifying the English.

File: docs/ibexa_products/headless.md [hunk 1]
```diff
@@ -119,11 +119,12 @@ Like everything in the Back Office, the calendar is extendable: you can add an e
 #### Many ways to structure and organize content
 
 [Product Information Management](pim_guide.md) (PIM) helps organize complex products and their catalogs:
-    - Product are organized per product types × variants × catalogs × categories × tags.
-    - Product attributes are grouped and factorized among product types. For example, fabric + color + size can be shared by many clothing product types.
-    - Product variants can rapidly be created by the automatic declination of attributes having a defined set of values.
+
+- Product are organized per product types × variants × catalogs × categories × tags.
```
Comment (dabrt) [inline docs/ibexa_products/headless.md:123]:
```suggestion
- Products are organized into product types, variants, catalogs, categories, and tags.
```

File: docs/ibexa_products/headless.md [hunk 2]
```diff
@@ -119,11 +119,12 @@ Like everything in the Back Office, the calendar is extendable: you can add an e
...
+- Product attributes are grouped and factorized among product types. For example, fabric + color + size can be shared by many clothing product types.
+- Product variants can rapidly be created by the automatic declination of attributes having a defined set of values.
```
Comment (dabrt) [inline docs/ibexa_products/headless.md:125]:
```suggestion
- Product variants can rapidly be created by the automatic declination of attributes that have a defined set of values.
```

**Review by julitafalcondusza — APPROVED** [review]

File: docs/ibexa_products/headless.md [hunk 3]
```diff
@@ -119,11 +119,12 @@ Like everything in the Back Office, the calendar is extendable: you can add an e
...
 - With taxonomy, you can tag content items to organize them by topics in a much intuitive way for the editor than a content tree with multiple locations would.
-  Tags themselves are organized in a tree, and synonyms are linked to a favorite terms. This tags organization can be the task of a supervisor who won't need to move content items around a corporate content tree. At search time, tags can be keywords with a high value in relevance score to help the end user having results closer to the searched topic.
+  Tags themselves are organized in a tree, and synonyms are linked to favorite terms. This tags organization can be the task of a supervisor who won't need to move content items around a corporate content tree. At search time, tags can be keywords with a high value in relevance score to help the end user having results closer to the searched topic.
```
Comment (dabrt) [inline docs/ibexa_products/headless.md:127]:
```suggestion
Tags themselves are organized in a tree, and synonyms are linked to favorite terms.
Tag organization can be handled by a supervisor who won't need to move content items around a corporate content tree.
At search time, tags can be keywords with a high value in relevance score to help the end user having results closer to the searched topic.
```

**Review by dabrt — APPROVED** [review]

**Pull request merged**